manager/integration/tests/test_scheduling.py

```diff
@@ -611,18 +611,12 @@ def test_data_locality_basic(client, core_api, volume_name, pod, settings_reset)
     pod1['spec']['nodeSelector'] = \
         {"kubernetes.io/hostname": volume1_attached_node}
     create_and_wait_pod(core_api, pod1)
-
-    wait_for_rebuild_start(client, volume1_name)
-
-    volume1 = client.by_id_volume(volume1_name)
-    assert len(volume1.replicas) == 2
-
-    wait_for_rebuild_complete(client, volume1_name)
-
```

Comment:
We can miss the rebuilding event if the pod takes a bit longer to come up. Moreover, making sure that there is exactly a local replica in the below check is enough. Therefore, skip this check.

Reply:
Thank you! Fixed it